assets/src/components/propertiesPanel/header.tsx

```diff
@@ -121,7 +141,30 @@ const directionName = (
 
 const nowInSeconds = (): number => Math.floor(Date.now() / 1000)
 
-const Header = ({ vehicle, route }: Props) => {
+const Tab = ({
```

Comment:
Could a lot of this live in `tabPanels.tsx` instead of here? The content half has a nicely abstracted component, and I think the button half would benefit from the same.

Reply:
Yeah I think the thing to do here would be to extract a third file, `tab.tsx`